src/main/java/com/minecolonies/colony/jobs/JobBuilder.java

```diff
@@ -2,6 +2,8 @@
 
 import com.minecolonies.client.render.RenderBipedCitizen;
 import com.minecolonies.colony.CitizenData;
+import com.minecolonies.colony.buildings.AbstractBuilding;
+import com.minecolonies.colony.workorders.AbstractWorkOrder;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.colony.workorders.AbstractWorkOrder'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)